images/vscode-ssh-server/README.md

```diff
@@ -0,0 +1,9 @@
+# Marinated Concrete's VSCode-compatible ssh server
+This runs a little ssh server that's compatible with VSCode's remote SSH feature. Unfortunately, [the LinuxServer one](https://docs.linuxserver.io/images/docker-openssh-server/) cannot be used because it is based on Alpine so it lacks glibc, a [known limitation](https://code.visualstudio.com/docs/remote/ssh#_remote-ssh-limitations).
+
+It's also less flexible/more opinionated.
+
+Configuration:
+- Set the environment variable `AUTHORIZED_KEYS_URL` to a path containing your authorized keys. For instance, `https://github.com/$GH_USERNAME.keys`.
+- Mount a small empty dir in `/config` - this will container your server's ssh host key, which must be persisted
```

Comment:
Would it be better if this was mounted via a `secret`?

Reply:
The intention is that you can do either - bring your own host keys, or let it autogenerate and persist them. I don't see value for my own use cases to bring my own, but it's no extra effort to support it.

Reply:
I added an example to show both options